### PR TITLE
 Documentation: add PD to production users

### PR DESCRIPTION
**Pull request by siddontang**
PTAL @xiang90

### CODE REVIEW

**Comment by heyitsanthony** [comment]
@siddontang please update the commit title to `Documentation: add PD to production users` so the commit conforms with https://github.com/coreos/etcd/blob/master/CONTRIBUTING.md#format-of-the-commit-message. Thanks!

**Comment by heyitsanthony** [comment]
@siddontang I meant the title / first line in the commit itself-- change with `git rebase -i master` and squash into a single commit? Thanks!

**Comment by siddontang** [comment]
@heyitsanthony Done :-)

File: Documentation/production-users.md [hunk 1]
```diff
@@ -67,5 +67,17 @@ Radius Intelligence uses Kubernetes running CoreOS to containerize and scale int
...
+
+- *Application*: embed etcd  
+- *Launched*: Mar 2016
+- *Cluster Size*: 3 or 5 members 
+- *Order of Data Size*: megabytes
+- *Operator*: PingCAP, Inc.
+- *Environment*: Bare Metal, AWS, etc.
+- *Backups*: None.
+
+PD(Placement Driver) is the central controller in the TiDB cluster. It saves the cluster meta inforamtion, schedule the data, allocate the global unique timestamp for the distrubted transaction, etc. It embeds etcd to supply high availability and auto failover.
```
Comment (heyitsanthony) [inline Documentation/production-users.md]: s/inforamtion/information
s/distrubted/distributed

**Comment by heyitsanthony** [comment]
Caught some typos. Should be good to merge after the fixup. Sorry about the back and forth. Thanks again!

**Comment by siddontang** [comment]
Thanks @heyitsanthony

**Comment by xiang90** [comment]
LGTM. Thanks.

**Pull request merged**